recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -107,16 +107,34 @@ def fail_if_travis_not_allowed_for_arch(config_file, feedstock_root):
         raise RuntimeError("Travis CI cannot be used on x86_64 in conda-forge!")
 
 
+def maybe_use_dot_conda(feedstock_root):
+    """Maybe set the .condarc to use .conda files."""
+    if os.path.exists(os.path.join(feedstock_root, "conda-forge.yml")):
+        with open(os.path.join(feedstock_root, "conda-forge.yml")) as f:
+            repo_config = safe_load(f)
+        pkg_format = repo_config.get("conda_pkg_format", None)
+        if pkg_format is not None:
+            try:
+                call([
+                    "conda", "config", "--env", "--set",
+                    "conda_build.pkg_format", str(pkg_format)
+                ])
+            except subprocess.CalledProcessError:
+                pass
```

Comment:
Why is it allowed to pass?

Reply:
Well, do we want to stop a feedstock from building due to an error here?

Reply:
For the record, we might. I wasn't sure on the first pass though since the old artifact style still works.

Reply:
Yes?! Why not? We'd want to build with the expected config (and be informed if it doesn't work), don't we?

Reply:
> For the record, we might. I wasn't sure on the first pass though since the old artifact style still works.

Ah, if this is meant as a temporary thing until we are sure all things are working, then that's fine (needs a `TODO`/`FIXME` comment then though ;P ).

Reply:
fixed! thanks for the review!